BUILD

```diff
@@ -259,6 +262,25 @@ cc_proto_library(
 )
 
 ################################################################################
+# Well Known Types Proto Library Rules
+################################################################################
+
+internal_copied_filegroup(
+    name = "_internal_wkt_protos",
+    srcs = WELL_KNOWN_PROTOS,
+    dest = "",
+    strip_prefix = "src",
+    visibility = ["//visibility:hidden"],
+)
+
```

Comment:
Can you add a comment to explain how to use these protos? An example should be good enough.

Reply:
Done